src/RapidezStatamicServiceProvider.php

```diff
@@ -32,9 +31,7 @@ class RapidezStatamicServiceProvider extends ServiceProvider
 {
     public function register()
     {
-        $this->app->extend(Sites::class, function () {
-            return new SitesLinkedToMagentoStores(config('statamic.sites'));
-        });
+        $this->app->extend(Sites::class, fn () => new SitesLinkedToMagentoStores());
```

Comment:
Isn't `$this->app->extend(Sites::class, SitesLinkedToMagentoStores::class);` enough?

Reply:
No the second parameter had to be a closure, keeping it on this for now